aws/resource_aws_api_gateway_method.go

```diff
@@ -51,6 +51,12 @@ func resourceAwsApiGatewayMethod() *schema.Resource {
 				Optional: true,
 			},
 
+			"authorization_scopes": &schema.Schema{
+				Type:     schema.TypeList,
```

Comment:
Does the ordering matter for this attribute? If not, we can use `schema.TypeSet` which has some nice helpers for doing set differences. See also: https://www.terraform.io/docs/extend/schemas/schema-types.html#typeset